src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -750,3 +771,16 @@ def upload_files_helper(
         client=client,
         test_id=test_id, yaml_data=yaml_data, test_plan=test_plan,
         load_test_config_file=load_test_config_file, existing_test_files=files, wait=wait)
+
+
+def validate_engine_data_with_regionwiseload_data(engine_instances, regionwise_engines):
+    if regionwise_engines is None:
+        return
+    total_engines = 0
```

Comment:
Is there any utility in azure_core that we can use to validate region names as well ?

Reply:
We had a discussion about this, it was considered not in scope on 2 regards, 1. region would need to be a valid region from ALT service perspective as well. 2. portal also doesn't do it on the fly, its hardcoded in the UI side afaik, and API is able to handle the invalid region.

However, I am checking with azure team on this, if I find something, I'll take it up at a later point. Anyways, the most it would verify is whether the name string is correct, it would not check whether it's a valid region for ALT